wifiphisher/pywifiphisher.py

```diff
@@ -538,7 +539,8 @@ def start(self):
                 if self.network_manager.is_interface_valid(
                         args.internetinterface, "internet"):
                     internet_interface = args.internetinterface
-                    self.network_manager.unblock_interface(internet_interface)
+                    if pyw.iswireless(internet_interface):
+                        self.network_manager.unblock_interface(internet_interface)
```

Comment:
Can you re-write this part? To keep things clear, it's better not to import `pyric` directly from the main engine but do all the operations through interfaces.py instead.